cockroachcloud/cluster-overview-page.md

```diff
@@ -20,17 +20,17 @@ The **Cluster settings** panel displays the settings you chose during [cluster c
 
 | Field     | Description                                                                                             |
 |-----------|---------------------------------------------------------------------------------------------------------|
-| Cloud     | The cluster's [cloud provider](create-a-serverless-cluster.html#step-2-select-a-cloud-provider-region). |
+| Cloud     | The cluster's [cloud provider](create-a-serverless-cluster.html#step-2-select-the-cloud-provider). |
 | Plan type | The plan type used to create the cluster.                                                               |
-| Region    | The cluster's [region](create-a-serverless-cluster.html#step-2-select-a-cloud-provider-region).         |
+| Region(s)    | The cluster's [region(s)](create-a-serverless-cluster.html#step-3-select-the-region-s).         |
 
 ## Usage this month
 
 The **Usage this month** panel displays your cluster usage statistics for the current month. You can click [**Estimate usage cost**](serverless-cluster-management.html#estimate-usage-cost) to open a tool that estimates your monthly costs based on your workload during a selected time frame.
 
 | Field         | Description                                                                                                                                                                                                                                                                      |
 |---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Resource limits   | The resource limits you entered when [creating the cluster](create-a-serverless-cluster.html#step-3-set-resource-limits). Click the **edit** icon to change the resource limits.                                                                                                         |
+| Resource limits   | The resource limits you entered when [creating the cluster](create-a-serverless-cluster.html#step-4-set-resource-limits). Click the **edit** icon to change the resource limits.                                                                                                         |
```

Comment:
I don't love "you entered when creating the cluster". You might not have created it, the values might have already been edited, etc. What about just "The serverless cluster's configured resource limits"?